Tests/Core/CheckoutAPIClientTests.swift

```diff
@@ -98,6 +98,33 @@ final class CheckoutAPIClientTests: XCTestCase {
     
     // MARK: - createCardToken
     
+    func test_createCardToken_cardTokenRequest_createLoggerCalledWithCorrectCorrelationID() {
+        
+        let stubRequestExecutor = StubRequestExecutor<CkoCardTokenResponse>()
+        let subject = createSubject(requestExecutor: stubRequestExecutor)
+        
+        let expectedCorrelationID = "correlation_id"
+        stubCorrelationIDGenerator.generateCorrelationIDReturnValue = expectedCorrelationID
+        
+        let cardTokenRequest = CkoCardTokenRequest(number: "4242", expiryMonth: "1", expiryYear: "2038", cvv: "100")
+        subject.createCardToken(card: cardTokenRequest) { _ in }
+        
+        let actualCorrelationID = stubNetworkFlowLoggerProvider.createLoggerCalledWithCorrelationID
+        XCTAssertEqual(expectedCorrelationID, actualCorrelationID)
+    }
+    
+    func test_createCardToken_cardTokenRequest_createLoggerCalledWithCorrectTokenType() {
+        
+        let stubRequestExecutor = StubRequestExecutor<CkoCardTokenResponse>()
+        let subject = createSubject(requestExecutor: stubRequestExecutor)
+        stubCorrelationIDGenerator.generateCorrelationIDReturnValue = ""
+        
+        let cardTokenRequest = CkoCardTokenRequest(number: "4242", expiryMonth: "1", expiryYear: "2038", cvv: "100")
```

Comment:
I like the use of the end of time here 😂

Reply:
Fantastic spot! 😎